content/blog/2024-04-23-april-project-meeting-update/contents.lr

```diff
@@ -0,0 +1,23 @@
+title: Prism: InvenioRDM Partner Meeting Summary, March 2024
+---
+author: Sara Gonzales
+---
+pub_date: 2024-04-23
+---
+team: InvenioRDM
+---
+body:
+
+![](2024-03-18-22_InvenioRDM-Workshop_1.jpg)
+
+The InvenioRDM partner community met in Münster, Germany from March 18 - 22, 2024, for our first in-person annual meeting in over four years. Forty-six attendees from over fifteen institutions spent the five days making connections, planning, and diligently working on all things InvenioRDM.
+
+Organizer Sarah Wiechers, software developer and research data manager for the Service Center for Data Management, ULB Münster, proposed the use of [Open Space Technology](https://en.wikipedia.org/wiki/Open_space_technology), a meeting organization principle in which the agenda and topics discussed are voted upon and implemented by the meeting’s attendees on the meeting date. This methodology was highly effective, allowing all community members the opportunity to pitch ideas and vote on their favorites for discussion.
+
+University of Münster’s [InvenioRDM Community](https://github.com/ulbmuenster/InvenioRDM-community) GitHub site contains our full output of discussions, decisions, and plans. Key topics addressed were the timely handling of pull requests, planning for the v12 release, improvements to community workflows, translations, large file management, deployment, digital preservation, vocabularies and fixtures, Kubernetes Helm-charts requirements, and much more. To help manage feature requests going forward, we have agreed to implement [GitHub Discussions](https://github.com/inveniosoftware/product-rdm/discussions/new?category=ideas), a tool which will allow for providing context, asking questions, and upvoting of most-wanted features. GitHub Discussions is now part of our workflow for keeping the [Roadmap](https://github.com/inveniosoftware/product-rdm?tab=readme-ov-file) updated.
+
+We were thrilled to be able to work with so many old and new friends in beautiful Münster, and a huge thanks goes to our local hosts at University of Münster for their hard work and hospitality. We hope that you will be able to join us at next year’s meeting, with Hamburg currently slated as the host city. In the meantime, please take advantage of our following updated community workflows, including:
+
+- Half of each telecon devoted to developers’ merge-focused chat
+- A new email list - coming soon
+- Consider joining an Interest Group (long-standing) or Task Group (deliverable-focused). For the current list, see the new [Onboarding page](https://inveniordm.docs.cern.ch/contribute/onboarding/)
```

Comment:
To keep noun-based parallelism + widen scope of developer chats to beyond just merging PRs:

```suggestion
- Development-focused chats during half of each telecon
- A new email list - coming soon
- Newly established Interest Groups (long-standing) or Task Groups (deliverable-focused). For the current list of all Groups, see the new [Onboarding page](https://inveniordm.docs.cern.ch/contribute/onboarding/)
```